test/cmd/builds.sh

```diff
@@ -12,16 +12,64 @@ os::log::install_errexit
 url=":${API_PORT:-8443}"
 project="$(oc project -q)"
 
+# expect_build_output_spec runs `oc new-build` with given arguments and expects
+# a build config with a given name to be created with a given build output spec
+function os::cmd::builds::expect_build_output_spec() {
```

Comment:
Very soon (next week?) I will be tweaking the output of the functions to read `(Running|SUCCESS|FAILURE): file:line_number: wrapper args`, where `file:line_number:` will be the place where the wrapper is called. For this case, the failure text will bring you to the call inside the function when instead you want the call of `os::cmd::builds::expect_build_output_spec`. You could still see that from the stack trace, but I'm not sure if it's worth the confusion - especially since jUnit integration will use that line number and file to track flakes, we won't get useful data for flakes using this function.